tests/testsuite/artifact_dep.rs

```diff
@@ -673,7 +703,22 @@ fn build_script_with_bin_artifact_and_lib_false() {
     p.cargo("build -Z bindeps")
         .masquerade_as_nightly_cargo(&["bindeps"])
         .with_status(101)
-        .with_stderr_does_not_contain("[..]sentinel[..]")
+        // This should not contain "sentinel"
+        .with_stderr_data(str![[r#"
```

Comment:
I'll want to look closer at this as comments could be missed in snapshot updates

Reply:
I can add back in explicit `.with_stderr_does_not_contain` if you would like

Reply:
Let's do that